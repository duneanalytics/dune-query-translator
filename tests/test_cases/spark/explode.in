select explode(column) as exploded from foo
