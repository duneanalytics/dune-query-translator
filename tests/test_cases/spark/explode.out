SELECT
  exploded
FROM foo
CROSS JOIN UNNEST(column) AS _u(exploded)